src/gc.rs

```diff
@@ -7,17 +7,19 @@
 #[derive(Debug, Eq, PartialEq, Clone, Copy)]
 pub struct GCType(u32);
 
-pub const GC_TYPE_TYPE_SCAVENGE: GCType = GCType(1);
+impl GCType {
+  pub const SCAVENGE: Self = Self(1 << 0);
 
-pub const GC_TYPE_MINOR_MARK_COMPACT: GCType = GCType(2);
+  pub const MINOR_MARK_COMPACT: Self = Self(1 << 1);
 
-pub const GC_TYPE_MARK_SWEEP_COMPACT: GCType = GCType(4);
+  pub const MARK_SWEEP_COMPACT: Self = Self(1 << 2);
 
-pub const GC_TYPE_INCREMENTAL_MARKING: GCType = GCType(8);
+  pub const INCREMENTAL_MARKING: Self = Self(1 << 3);
 
-pub const GC_TYPE_PROCESS_WEAK_CALLBACK: GCType = GCType(16);
+  pub const PROCESS_WEAK_CALLBACKS: Self = Self(1 << 4);
 
```

Comment:
for these flags is there a way we could assert they're the same as in v8?

Reply:
Not sure - what do you suggest?

These are tested in test_api.rs in the sense that they are used and we check they behave correctly. Is that enough?

Reply:
nevermind - I can't think of a way to assert them being the same as in C